lib/feature_flagger/manifest_sources/with_yaml_file.rb

```diff
@@ -7,7 +7,7 @@ def initialize(yaml_path = nil)
       end
 
       def resolved_info
-        @resolved_info ||= ::YAML.load_file(@yaml_path) if @yaml_path
+        @resolved_info ||= ::YAML.unsafe_load_file(@yaml_path) if @yaml_path
```

Comment:
Instead of using unsafe_load_file, you should pass down the permitted classes you can safely load from.

```suggestion
        @resolved_info ||= ::YAML.oad_file(@yaml_path, permitted_classes: [Date]) if @yaml_path
```

ex:
```ruby
hash = YAML.load_file(
  some_file_name, 
  permitted_classes: [Matrix, OpenStruct, Symbol, Date]
)
```